packages/amplify-graphql-relational-transformer/src/utils.ts

```diff
@@ -263,6 +411,17 @@ export const getFieldsNodes = (
     return fieldNode;
   });
 };
+const getAssociatedRelationalDirectiveTypes = (sourceRelationalDirectiveType: string): string[] => {
+  switch (sourceRelationalDirectiveType) {
+    case 'hasOne':
```

Comment:
nit: Can we use the names from the `amplify-graphql-directives` package, rather than hardcoding them?

Reply:
Thanks for the suggestion, addressed in [8e5ec28](https://github.com/aws-amplify/amplify-category-api/pull/2482/commits/8e5ec2840e675eb5dc81633b6614adf11a598dba)